packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -501,11 +502,24 @@ const Dashboard: FC = () => {
                         isEditMode={isEditMode}
                     />
                 )}
+
+                {isDeleteModalOpen && (
+                    <DashboardDeleteModal
+                        opened
+                        uuid={dashboard.uuid}
+                        onClose={() => setIsDeleteModalOpen(false)}
+                        onConfirm={() => {
+                            deleteDashboard(dashboard.uuid);
```

Comment:
We already delete the dashboard in the model:
```
    const handleConfirm = async () => {
        await deleteDashboard(uuid);
        onConfirm?.();
    };
```
So this is going to throw an error.